addons/binding/org.openhab.binding.homematic/OSGI-INF/HomegearBridgeDiscovery.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.smarthome.binding.homematic.discovery.bridge">
+   <implementation class="org.openhab.binding.homematic.discovery.HomegearDiscoveryParticipant"/>
```

Comment:
It is only homegear that can be discovered and not a CCU? That's really a feature that eQ-3 should add to it...